### DISCUSSION TITLE
Android: Buttons in merge PR bottom sheet partially/not visible for very long commit messages

### DISCUSSION THREAD

**Discussion by svenjacobs** [community/community#69545] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I noticed that the buttons in the merge pull request bottom sheet are either only partially or not visible at all when the commit message is very long, like messages created by Dependabot.

It seems the commit message pushes down other UI elements when it grows but the sheet's contents is not scrollable, so the buttons cannot be reached.

App version: 1.133.0
Device: Google Pixel 7a
Android version: 14 (UP1A.231005.007)

![Screenshot_20231007-103946](https://github.com/community/community/assets/255313/8bce44d5-de9c-44b0-a2e6-669d27b9612d)

![Screenshot_20231007-104001](https://github.com/community/community/assets/255313/82446247-b6d4-4563-bdff-535451ad08cd)

**Comment by alcere** [comment, marked as answer]
Hi @svenjacobs. Thank you very much for reporting this ugly issue, we are deeply sorry for the troubles here! 
We have created a fix for this that will be shipped tomorrow. Thanks again!

**Reply by svenjacobs** [reply]
Thanks for the quick fix! 🎉

**Reply by karfau** [reply]
Has the fix already been shipped? I don't see an update yet, but I'm also effected by that issue now.

**Reply by alcere** [reply]
Hi there! We shipped the fix yesterday to our beta audience and it will be rolling out today for the rest of our users. We appreciate your patience and apologize for any inconvenience this may have caused.
